README.md

```diff
@@ -97,6 +97,11 @@ This option can also be set using `-alertmanager.ttl` flag. Example:
 
 Default is `1m`.
 
+Note that maximum value for this option is `15m`.
+UI have a watchdog feature that will track the timestamp of last pull, if UI
+receiving updates for more than 15 minutes it will print an error and
```

Comment:
Should this be "not receiving updates"? If so, make that sentence "If the UI does not receive updates for more than 15 minutes".